chat/fe/src/views/RoomView.ts

```diff
@@ -110,9 +165,113 @@ export function RoomView(props: RoomViewProps) {
       messageContentEl.append(messageTime);
 
       const bodyEl = Div();
+      setStyle(bodyEl, {
+        position: 'relative',
+      });
 
       bodyEl.innerHTML = autolinker.link(props.body);
 
+      const currentUser = getSelf();
+      if (props.user === currentUser._id) {
+        const messageOptions = Div({
+          class: 'options-button',
+        });
+        setStyle(messageOptions, {
+          position: 'absolute',
+          top: '-18px',
+          right: '0px',
+          display: 'none',
+          cursor: 'pointer',
+          width: '24px',
+          height: '12px',
+          paddingBottom: '10px',
+          textAlign: 'center',
+          fontSize: '14px',
+          border: '1px solid #909090bf',
+          backgroundColor: '#f2f2f2',
+          color: '#909090',
+          borderRadius: '2px',
+        });
+
+        messageOptions.innerHTML = '&#8230';
```

Comment:
Is it possible to just paste the unicode character?  I'm not sure whether innerHTML allows or not, but that would make it easier to know what character this is

Reply:
yeah, it's this for future reference: \u2026